ait/core/server/broker.py

```diff
@@ -25,6 +25,12 @@ def __init__(self):
         self.XPUB_URL = ait.config.get('server.xpub',
                                         ait.SERVER_DEFAULT_XPUB_URL)
 
+        ## Name of the topic associated with external commands
+        self.command_topic = ait.config.get('command.topic',
+                                        ait.DEFAULT_CMD_TOPIC)
+        ## Controls if server selects a default if one not speicified
+        self.command_stream_select_defaut = True
```

Comment:
This isn't configurable so I vote we drop this. Worst case we can add it similar functionality in the future if this is a use case we want to cover (currently I don't think it is)